src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ColorSupplier randomColour = new ColorSupplier();
+        Random random = new Random();
+
+        ball.setColour(randomColour.getRandomColor());
+        ball.setNumber(random.nextInt(100));

```

Comment:
you need to define a constant for `100`

something like `private static final int MAX_BALL_NUMBER = 100`;